apps/wallet/src/ui/app/shared/faucet/useFaucetMutation.ts

```diff
@@ -29,7 +38,7 @@ export function useFaucetMutation(options?: UseFaucetMutationOptions) {
 	return {
 		...mutation,
 		/** If the currently-configured endpoint supports faucet: */
-		enabled: !!api.connection.faucet,
+		enabled: !!options?.host,
```

Comment:
this is probably also not the right way to do this.... we might need to rework this entire hook (although maybe setting up a faucetProvider or something might work 🤷

Reply:
I think this is fine fwiw